#Массив значений протечек на аппарате
uint8[8] status